lib/file-perms.c

```diff
@@ -1,6 +1,6 @@
 /*
  * Copyright (c) 2012 BalaBit IT Ltd, Budapest, Hungary
```

Comment:
@bazsi I've submitted the whole thing as a separate PR. I include the script used for transformation when a commit is not done manually.
https://github.com/balabit/syslog-ng/pull/823